<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SariSariSleuth</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Sales</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="openCameraButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Open Camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="manualAddButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Manual Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Transactions</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableView" name="transactionTableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteTransactionButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>Stock Logging</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filterLineEdit">
          <property name="placeholderText">
           <string>Search products...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="stockTableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>Tab 4</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushSales">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	color:white;
	background-color:rgb(244, 98, 98);
	font: 700 20pt &quot;Arial&quot;;
	border-radius: 30px;
}

QPushButton:Hover{
	background-color:rgb(179, 69, 69);
}

QPushButton:Pressed{
	background-color:rgb(179, 69, 69);
}</string>
    </property>
    <property name="text">
     <string>SALES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushTransactions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	color:white;
	background-color:rgb(244, 98, 98);
	font: 700 17pt &quot;Arial&quot;;
	border-radius: 30px;
}

QPushButton:Hover{
	background-color:rgb(179, 69, 69);
}

QPushButton:pressed{
	background-color:rgb(179, 69, 69);
}</string>
    </property>
    <property name="text">
     <string>TRANSACTIONS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStock">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	color:white;
	background-color:rgb(244, 98, 98);
	font: 700 20pt &quot;Arial&quot;;
	border-radius: 30px;
}

QPushButton:Hover{
	background-color:rgb(179, 69, 69);
}

QPushButton:Pressed{
	background-color:rgb(179, 69, 69);
}</string>
    </property>
    <property name="text">
     <string>STOCK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAnalytics">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	color:white;
	background-color:rgb(244, 98, 98);
	font: 700 20pt &quot;Arial&quot;;
	border-radius: 30px;
}

QPushButton:Hover{
	background-color:rgb(179, 69, 69);
}

QPushButton:Pressed{
	background-color:rgb(179, 69, 69);
}</string>
    </property>
    <property name="text">
     <string>ANALYTICS</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 69, 69);
border-radius: 20px</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pushSales</zorder>
   <zorder>pushTransactions</zorder>
   <zorder>pushStock</zorder>
   <zorder>pushAnalytics</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
